Looks like the test-case is failing, and needs to be updated:

```
--- FAIL: TestTemplate (0.00s)
    template_test.go:14: unexpected template size 2645 != 2766
```

--- a/processor/emailer/emailer.go
+++ b/processor/emailer/emailer.go
@@ -165,7 +165,7 @@ func encodeHeader(s string) string {
 	return "=?utf-8?Q?" + se + "?="
 }
 
-// Encode email header entry to comply List-ID restriction of RFC 2919
+// makeListIdHeader encodes email header entry to comply List-ID restriction of RFC 2919
 // according to DRUMS.
 //
 // The function has some special code to handle an URL.

--- a/template/template_test.go
+++ b/template/template_test.go
@@ -8,7 +8,7 @@ func TestTemplate(t *testing.T) {
 
 	// content and expected length
 	content := EmailTemplate()
-	length := 2645
+	length := 2766
 
 	if len(content) != length {
 		t.Fatalf("unexpected template size %d != %d", length, len(content))

